Server/controllers/monitorController.js

```
@@ -177,8 +183,7 @@ const getMonitorsAndSummaryByTeamId = async (req, res, next) => {
   } catch (error) {
     error.status = 422;
     error.service = SERVICE_NAME;
-    error.method === undefined &&
-      error.method === "getMonitorsAndSummaryByTeamId";
+    error.method === undefined? error.method = "getMonitorsAndSummaryByTeamId": null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)